scripts/_common.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
 
+#=================================================
+# COMMON VARIABLES
+#=================================================
+
+# dependencies used by the app
+pkg_dependencies="postgresql libstdc++6 redis-server rabbitmq-server libcurl4-nss-dev"
+
 #=================================================
 # PERSONAL HELPERS
 #=================================================
-source="https://github.com/ONLYOFFICE/DocumentServer/releases/download/ONLYOFFICE-DocumentServer-5.2.8/onlyoffice-documentserver_amd64.deb"
-version="5.2.8"
+
+onlyoffice_version="5.2.8"
+
+onlyoffice_source="https://github.com/ONLYOFFICE/DocumentServer/releases/download/ONLYOFFICE-DocumentServer-$onlyoffice_version/onlyoffice-documentserver_amd64.deb"
```

Comment:
By the way, as you now use apt, do you still need that variable ?

Reply:
Old variables removed. I forgot to remove them at the end of the packaging.